.coveragerc

```diff
@@ -459,6 +459,7 @@ omit =
     homeassistant/components/*/spider.py
 
     homeassistant/components/air_quality/opensensemap.py
+    homeassistant/components/air_quality/waqi.py
```

Comment:
Can you move the file to `homeassistant/components/waqi/air_quality.py` ? (and add an empty `waqi/__init__.py` file )

Reply:
Sorry, I must be confused but I do not understand why. Shouldn't air_quality components like those live in `homeassistant/components/air_quality/`? Like the current `opensensemap`.

Reply:
Disregard that, I wasn't up to date on the more recent organization changes.

Reply:
I like this change, I have multiple components with at least two platforms, this will make things easier.